src/components/ReplayForm.vue

```diff
@@ -222,6 +246,36 @@ ${boPaLabel} ${expectedGamesCount.value}
 Map draft: ${extractDraftUrl(mapDraft.value)}
 Civ draft: ${extractDraftUrl(civDraft.value)}`
 })
+
+function updateMeta(newErrors: ReplayErrors, newMeta: ReplayMetadata) {
+  if (!props.tournament && newMeta?.maps?.pickedMaps?.length) {
+    const numOfMaps = newMeta.maps.pickedMaps.length
+    if (numOfMaps % 2 == 0) {
+      setExpectedGamesCount(numOfMaps + 1)
+    } else {
+      setExpectedGamesCount(numOfMaps)
+    }
+  }
+  meta.value = newMeta
+  metaErrors.value = newErrors
+  if (!props.tournament || !meta.value.maps?.preset) {
+    return
+  }
+  const presetToSetType = Object.fromEntries(
+    Object.entries(props.tournament.maps).map(([setType, preset]) => [
+      preset,
+      expandSetAbbreviation(setType)
+    ])
+  )
+  if (!(meta.value.maps.preset in presetToSetType)) {
+    console.error('Map preset is not in the tournament set, but it should')
```

Comment:
This should not happen, since it would be an error to have a draft that is not from the tournament's presets